### PR TITLE
Fix/small UI tweaks

### LINKED ISSUE #11: Disable text selection
Disable Text Selection in main screen (e.g. Kapta by UCL; or the instructions)

### PR DESCRIPTION
**Pull request by acholyn**
addresses #5 
closes #11 & #7

### CODE REVIEW

**Review by MarcosMoreu — COMMENTED** [review]
I haven't installed it with this fix. Just checking if replacing the icons has fixed the problem of wrong size/display in the homescreen & splash logo once installed??

**Review by MarcosMoreu — APPROVED** [review]

**Review by MarcosMoreu — APPROVED** [review]

File: src/styles/main.css [hunk 1]
```diff
@@ -76,7 +77,7 @@ body {
         background-color: transparent;
         color: white;
         border: none;
-        font-size: 105%;
+        font-size: 108%;
```
Comment (MarcosMoreu) [inline src/styles/main.css:81]: Looks that it doesn't fit in a/some mobile screen. Keep it as 106 with the emojis in the next line (so emojis in between the text lines). Makes sense?

File: src/styles/main.css [hunk 2]
```diff
@@ -87,10 +88,10 @@ body {
 
     #instructions {
         text-align: center;
-        font-size: 105%;
+        font-size: 150%;
```
Comment (MarcosMoreu) [inline src/styles/main.css:92]: I just tested it and with this font doesn't fit to some mobile screen. Let's use 120%.

**Review by MarcosMoreu — CHANGES_REQUESTED** [review]
Still with this icons the maskable property in the manifest doesn't seem to work properly. The easiest solution is to use the previous icons with some margin. I've tried and it just works.

Replace with the attached 
[resized_images_new_3.zip](https://github.com/user-attachments/files/16354332/resized_images_new_3.zip)

**Review by MarcosMoreu — APPROVED** [review]

**Comment by acholyn** [comment]
video can be made full screen by clicking the option but the modal should sit just below the "how to" title 
image

**Pull request merged**